compiler+runtime/include/cpp/jank/runtime/detail/type.hpp

```diff
@@ -12,6 +12,7 @@
 
 namespace jank::runtime::detail
 {
+  bool equal(char const lhs, object_ptr const rhs);
```

Comment:
Please use `native_bool`. We have this alias in case we ever change the bool type. We'll be able to change it through the entire compiler and runtime using one line.